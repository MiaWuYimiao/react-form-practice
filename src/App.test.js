import { render, screen } from '@testing-library/react';
import App from './App';


test("renders without crash", function() {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});