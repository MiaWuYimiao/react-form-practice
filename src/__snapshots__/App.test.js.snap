// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crash 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <div>
        laundary
        <button>
          X
        </button>
      </div>
      <div>
        send an email
        <button>
          X
        </button>
      </div>
      <form>
        <label
          for="task"
        >
          Task: 
        </label>
        <input
          id="task"
          name="task"
          placeholder="task"
          type="text"
          value=""
        />
        <button>
          Add Todo
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
